website/src/components/playground.js

```diff
@@ -11,7 +11,7 @@ const getIframeUrl = (urlPath, isFullScreen = false) => {
 };
 
 const PlaygroundController = (props) => {
-  const [iframeHeight, setIframeHeight] = useState(0);
+  const [iframeHeight, setIframeHeight] = useState(400);
```

Comment:
This is a temporary fix to show the playground component, since the implementation expects the iframe to set the height.

Somehow the event sent via `postMessage` is not received by the parent. We'll need to find a better solution then as a follow up.